doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,144 @@
+[[renaming-ha]]
+== Updating Hostnames on Database-Only Appliances
+
+When updating the hostnames of database-only appliances in a cluster, you must also re-configure high availability on the primary and standby database-only appliances.
+
+=== Preparing to Update Appliance Hostnames
+
+Before updating the hostnames of your appliances, complete the following:
+
+. Note the hostnames of the active database (primary) appliance and the standby database appliances in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. On the {product-title_short} (non-database) appliances, stop the failover monitor: 
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. On the _standby_ database-only appliances, stop the `postgresql` database service:
++
+----
+# systemctl stop rh-postgresql95-postgresql
+----
++
+. Stop `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl stop evmserverd
+----
+
+
+=== Updating the Primary Database-Only Appliance Hostname
+
+Run the following steps on your primary database-only appliance:
+
+. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+. Restart the appliance.
+. Update the `host` key to the primary database appliance's new hostname in `/var/www/miq/vmdb/config/database.yml`.
+
+
+=== Updating the Standby Database-Only Appliance Hostname
+
+Run the following steps on your standby database-only appliances:
+
+. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+. Restart the appliance.
```

Comment:
Do we want to direct users to a command in the appliance_console or is this a cli command?

Reply:
It's an option in the appliance console menu. Just checking other places in the HA guide, looks like we made that clearer by writing it like "In the appliance console menu,...". So I'll change it to that. What do you think?

Reply:
Works for me, thanks